src/locales/en.json

```diff
@@ -150,6 +151,11 @@
     "lineEditor_pointSelected": "Press Delete to remove point, CtrlOrCmd+D to duplicate, or drag to move",
     "lineEditor_nothingSelected": "Select a point to move or remove, or hold Alt and click to add new points"
   },
+  "errorCanvasPreview": {
+    "title": "Cannot show preview",
+    "description": "The canvas may be too big.",
```

Comment:
This and line 124 are very similar.. can we get rid of one of them?

Reply:
Makes it easier for all the translations.

Reply:
Makes sense